src/components/containers/ExperimentContainer.js

```diff
@@ -79,12 +79,14 @@ class ExperimentContainer extends React.Component {
         } else if (experimentFetch.rejected) {
             return <Error message={experimentFetch.reason.message} />;
         } else if (experimentFetch.fulfilled) {
-            let visibleMetricIds;
+            let visibleMetricIds, searchActive;
```

Comment:
JIT prefers initialization.

Reply:
Ah sorry should've been clearer. Providing an initial value so `let visibleMetricIds = [], searchActive = false` for example.

Reply:
Gotcha. Updated. Do you also have a link to more info about that?

Reply:
I had a better one that I can no longer find but this explains it wrt to V8: https://www.html5rocks.com/en/tutorials/speed/v8/#toc-topic-numbers